installer/DetailsHandler.php

```diff
@@ -56,17 +56,177 @@ public function checkDetailsValidation(){
  		$companyName = $configurationDataSet["CompanyName"];
  		$wantSendData = $configurationDataSet["SendUsageDataToOrangeHRM"];
 		$isPort = $configurationDataSet["IsPort"];
-			
-		$this->setConfigurationFromParameter($dbHostName,$dbHostPort,$dbName,$adminUserName,$adminPassword,$dbOHRMUserName,$dbOHRMPassword,$dbUserName ,$dbPassword,$databaseRootPassword, $encryption , $dbCreateMethod , $sameOhrmUser,$companyName,$wantSendData, $isPort);
+
+    	$organizationName = $configurationDataSet["organizationName"];
+    	$adminEmployeeFirstName = $configurationDataSet["adminEmployeeFirstName"];
+    	$adminEmployeeLastName = $configurationDataSet["adminEmployeeLastName"];
+    	$organizationEmailAddress = $configurationDataSet["organizationEmailAddress"];
+
+		$this->setConfigurationFromParameter($dbHostName,$dbHostPort,$dbName,$adminUserName,$adminPassword,$dbOHRMUserName,$dbOHRMPassword,$dbUserName ,$dbPassword,$databaseRootPassword, $encryption , $dbCreateMethod , $sameOhrmUser,$companyName,$wantSendData, $isPort, $organizationName, $adminEmployeeFirstName, $adminEmployeeLastName, $organizationEmailAddress);
+}
+
+    /**
+	 * Get the organization name
+     * @return mixed
+     */
+public function getOrganizationName() {
+    $configurationDataSet = parse_ini_file("config.ini");
+
+    if($_SESSION['defUser']['organizationName']) {
+    	return $_SESSION['defUser']['organizationName'];
+	}
+    return $configurationDataSet["organizationName"];
+}
+
+    /**
+	 * Get the Admin Employee First name
+     * @return mixed
+     */
+public function getAdminEmployeeFirstName() {
+    $configurationDataSet = parse_ini_file("config.ini");
```

Comment:
This line is repeated everywhere.. better to add isCliInstaller private function and load the ini file once